### DISCUSSION TITLE
How to export the detection model with scores?

### DISCUSSION THREAD

**Discussion by Inkorak** [open-mmlab/mmdeploy#1865] [Q&A]
How to export the detection model with prediction scores?
Something like:
onnx_config = dict(
    dynamic_axes={
        'input': {
            0: 'batch',
            2: 'height',
            3: 'width'
        },
        'dets': {
            0: 'batch',
            1: 'num_dets',
        },
        'labels': {
            0: 'batch',
            1: 'num_dets',
        },
        'scores': {
            0: 'batch',
            1: 'num_dets',
        },
    })

**Comment by irexyc** [comment]
@Inkorak Hi, sorry for late reply, it's better to ask question in issues.

You can use deploy configs in `configs/mmdet/instance-seg` directory, which have mask output.
